modules/ROOT/pages/api-groups-add-sla-tiers.adoc

```diff
@@ -0,0 +1,30 @@
+= Add SLA Tiers To API Groups
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can add multiple SLA tiers to an API Group instance.
+
+To add an SLA tier:
+
+. From the left menu in API Manager, click *API Group*.
++
+The API Groups details page appears.
+. Expand the API Group list and click the API Group instance to which you want to add the SLA tier.
+. From the left menu, click *SLA Tiers* and then click *Add SLA Tiers* on the details page.
++
+. The illustration shows the Add SLA Tier window.
+image:api-groups-add-sla-tier.png[Add SLA Tier]
+. Click *Continue*.
++
+The Adding limits to SLA tier page appears.
+. At the top of the SLA tier details area, click `Default` if you want the SLA tier to apply to API instances in the API Group instance.
++
+Otherwise, click `Individual` to apply it to a specific API instance within the API Group instance.
+. In the `# of Reqs` field, add the number of request that are allowed, for example: `100`.
```

Comment:
I think it will be useful to explain how to copy an existing tier from the API

Reply:
Done! Thanks!